tap_mysql/sync_strategies/binlog.py

```diff
@@ -511,15 +539,18 @@ def sync_binlog_stream(mysql_conn, config, binlog_streams, state):
         reader = BinLogStreamReader(
             connection_settings={},
             server_id=server_id,
-            slave_uuid=f'stitch-slave-{server_id}',
+            slave_uuid=f'slave-{server_id}',
```

Comment:
if you want, you can rebrand it to `ppw-{server_id}` so people running `SHOW SLAVE HOSTS` or `SHOW REPLICA HOSTS` commands will see a bit more informations about the binlog consumer.

We do something similar in postgres [here](https://github.com/transferwise/pipelinewise/blob/5d4b4a432996cedbe9233946766e66fe5e44142d/pipelinewise/fastsync/commons/tap_postgres.py#L33) when creating replication slots and sometimes it helps to DBAs identifying consumers.

Reply:
Good call.

Reply:
Done https://github.com/transferwise/pipelinewise-tap-mysql/pull/57/commits/a96ae8123aeab052cf8dac691f3276da8caafc81